### PR TITLE
Deploy website with Realm

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

- Updated build command to generate static bundle, and outputs bundle to `hosting/files`
- Created website branch using `gh-pages`, which [uploads website directory](https://github.com/mongodb/leafygreen-ui/tree/website), including static build (which is ignored on master)
- Realm configured to automatically deploy any time there is an update to `website` branch
- Can see updates here: mongodb-design-egzxo.mongodbstitch.com/

🎟 _Jira ticket:_ [PD-780](https://jira.mongodb.org/browse/PD-780)

## 🛠 Types of changes

- [x] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

**Review by lazytype — APPROVED** [review]

File: website/package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && next export -o hosting/files/",
```
Comment (hswolff) [inline website/package.json:7]:
```suggestion
    "build": "next build && next export -o website/dist/",
```

Keep things standardized?

File: website/package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
...
+    "release:web": "gh-pages -b website --dist .",
```
Comment (hswolff) [inline website/package.json:8]: Do we intend on having multiple release namespaces? Could we just do `release`? 

Shame we can't do `publish`.

File: website/package.json (hunk 1 above)
Comment (brookescarlett) [reply website/package.json:7]: This needs to follow Realm's application configuration. That way when Realm deploys it knows where to look for the bundled files.

File: website/package.json (hunk 2 above)
Comment (brookescarlett) [reply website/package.json:8]: Yeah good call I can update this.

**Review by hswolff — APPROVED** [review]

**Pull request merged**